feat: redirect pages to a-propos and create page espace agent

XavierJp opened the pull request:
This PR groups several pages into an `/a-propos` route and creates a espace agent page

@renardpal @Jonathan2051 @johangirod what souhld we have in this information page ?


ToDo : 
- [x] add a CTA in header
- [x] fix the affordance issue of current `loginWall` component (add a more explicit explanation)

Moved to a dedicated PR #888
- [ ] espace agent page (should explian how it work, should have good SEO value)
- [ ] espace agent page can become a welcome page when connected

johangirod reviewed (approved):
I think it's good practice to limit the number of top level subpath, so I'm in favour !

Jonathan2051 commented:
I need more background information to answer now

XavierJp commented:
Johan's feedback : we must not assume the agents know they already have an account

johangirod commented:
Proposition de wording @renardpal @XavierJp 
![image](https://github.com/etalab/annuaire-entreprises-site/assets/1775934/bd1e3225-cc82-4c76-b7f9-72c657664d0d)

XavierJp commented:
Really like your suggestion @johangirod 

I also just improve hierarchy based on @renardpal recommandations : move the secondary explanation into a dedicated footer.

johangirod commented on components/espace-agent-components/agent-wall/index.tsx:
```diff
@@ -13,14 +14,21 @@ const AgentWall: React.FC<{
+            <div>
+              <h3>Vous êtes agent public ?</h3>
+              <p>
+                En cliquant sur le bouton ci-dessous, vous aurez un
```
```suggestion
                En cliquant sur le bouton ci-dessous, vous aurez
```

johangirod commented on components/espace-agent-components/agent-wall/index.tsx:
```diff
@@ -60,12 +68,21 @@ const AgentWall: React.FC<{
+
+        .cta-footer {
+          font-size: 0.9rem;
+          background-color: ${constants.colors.lightGrey};
```
On peut aussi utiliser celui du dsfr `var(--background-alt-grey)`

renardpal commented:
@johangirod très bonne suggestion d'avoir une phrase d'explication ! Je me permets de rewriter pour ajouter le lien agentconnect dans ce texte et pas sous le bouton et de faire officiellement de cette page une xp agentconnect, en simplifiant le bouton. qu'en penses-tu ? 

![image](https://github.com/etalab/annuaire-entreprises-site/assets/44638568/683ae842-f931-4b46-a62b-a860d861e9be)

renardpal commented:
Je voulais un peu supprimer le verbe parce que je trouve que c'est à moitié redondant avec le mot "agentconnect".
Mais testons en gardant un verbe d'action, auquel cas je préférerais m'en tenir au "s'identifier avec" dans le bouton

XavierJp commented:
@renardpal @johangirod 

- I renamed "Espace Agent Public" into "Compte agent public" as it is simpler and more accurate
- I delete the a propos page. I think we should rather add content to the `/connexion/agent-public`
- I added a CTA in header

<img width="1252" alt="image" src="https://github.com/etalab/annuaire-entreprises-site/assets/8900205/9daa7632-0e51-44ea-b56e-005a76b46385">

And 

<img width="1252" alt="image" src="https://github.com/etalab/annuaire-entreprises-site/assets/8900205/ea36220f-277a-487c-9ff1-44c68f5f17cd">

XavierJp commented:
@renardpal last commit is for you

XavierJp commented:
Added a carreer page link in blurred part
https://github.com/etalab/annuaire-entreprises-site/commit/a5ec1e30e99802642fc199e8d14e1338bb6ad2d5
